### PR TITLE
WT-8799 Disabled Documentation Build due to CMAKE changes on mongodb-5.0

### PR DESCRIPTION
**Pull request by clarissecheah**

### CODE REVIEW

**Review by etienneptl — APPROVED** [review]
LGTM!

File: test/evergreen.yml [hunk 1]
```diff
@@ -3941,6 +3941,9 @@ buildvariants:
     - name: package
 
 - name: documentation-update
+  # Documentation update is NOT expected to be triggerred on Evergreen projects
```
Comment (mm-ng) [inline test/evergreen.yml:3944]: Typo: should be `triggered`.

**Review by mm-ng — APPROVED** [review]
LGTM, with minor comment.

File: test/evergreen.yml (hunk 1 above)
Comment (clarissecheah) [reply test/evergreen.yml]: Fixed, thanks!

**Review by lukech — APPROVED** [review]
Changes LGTM. The solution is more of "disable doc build on mongodb-5.0". Can the PR/commit title be updated to reflect that?

**Pull request merged**